azure-cli/ad/deploy.swagger.ps1

```diff
@@ -0,0 +1,80 @@
+param (
+  [Parameter(Mandatory = $true)]
+  [ValidateNotNullOrEmpty()]
+  [string]
+  $companyHostName,
+
+  [Parameter(Mandatory = $true)]
+  [EnvironmentConfig] $environmentConfig,
+
+  [Parameter(Mandatory = $true)]
+  [NamingConfig] $namingConfig,
```

Comment:
Break line between type and parameter name

Reply:
Done